L1Trigger/TrackFindingTracklet/test/AnalyzerDemonstrator.cc

```diff
@@ -0,0 +1,192 @@
+#include "FWCore/Framework/interface/one/EDAnalyzer.h"
+#include "FWCore/Framework/interface/Run.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/EDGetToken.h"
+#include "FWCore/Utilities/interface/ESGetToken.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+#include "FWCore/Utilities/interface/Exception.h"
+#include "DataFormats/Common/interface/Handle.h"
+
+#include "L1Trigger/TrackerTFP/interface/Demonstrator.h"
+#include "L1Trigger/TrackFindingTracklet/interface/ChannelAssignment.h"
+
+#include <sstream>
+#include <utility>
+#include <numeric>
+
+using namespace std;
+using namespace edm;
+using namespace tt;
+using namespace trackerTFP;
+
+namespace trklet {
+
+  /*! \class  trklet::AnalyzerDemonstrator
+   *  \brief  Class to demontrate correctness of track trigger emulators
```

Comment:
Comment describing this class needs more info. e.g. How does it work? By comparing FW with SW?

Reply:
done